backend/corpora/dataset_processing/process_cxg.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+import logging
+
+from backend.corpora.common.utils.s3_buckets import s3_client
+from backend.corpora.dataset_processing.process import (
+    process_cxg,
+    get_bucket_prefix,
+)
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def download_from_s3(bucket_name: str, object_key: str, local_filename: str):
+    logger.info(f"Downloading file {local_filename} from bucket {bucket_name} with object key {object_key}")
```

Comment:
Is this the same download function from backend/corpora/dataset_processing/process_seurat.py? If so, please factor out.